etc/picongpu/bash-devServer-hzdr/gpu_v100_install.sh

```diff
@@ -1,10 +1,10 @@
-spack install --reuse cmake@3.24.2 %gcc@10.2.0
-spack load cmake@3.24.2 %gcc@10.2.0
+spack install --reuse cmake@3.26.3 %gcc@12.2.0
```

Comment:
You did not change gcc to version 12.1. here?

Reply:
fixed it